python/ray/_private/utils.py

```diff
@@ -78,6 +78,14 @@
 
 
 def get_user_temp_dir():
+    # When global mode is enabled in databricks runtime, ray.init() looks for
+    # existing ray cluster in corresponding default temp dir.
+    # This should be consistent with python/ray/util/spark/databricks_hook.py.
+    if (
+        os.environ.get("DATABRICKS_RAY_CLUSTER_GLOBAL_MODE", "false").lower() == "true"
+        and "DATABRICKS_RUNTIME_VERSION" in os.environ
+    ):
```

Comment:
```suggestion
    if (
        "DATABRICKS_RUNTIME_VERSION" in os.environ
    ):
```

For all cases in databricks runtime, we should use "/local_disk0/tmp" by default, for preventing /tmp disk running out of storage space